### PR TITLE
Answer:31 Module to standalone

### PR DESCRIPTION
**Pull request by LMFinney**
## Checklist for challenge submission

- [x] Start your PR message with Answer:${challenge_number}

## Warning:

- If you want feedback or review, you must support the project on GitHub:
  - [$5 per review](https://github.com/sponsors/tomalaforge)
  - [$25 for lifetime reviews](https://github.com/sponsors/tomalaforge)
  - Create one of more challenges
  - Have many valuable contributions to the project

Alternatively, you can still submit your PR to join the list of answered challenges or to be reviewed by a community member. 🔥

### CODE REVIEW

File: libs/module-to-standalone/shell/src/lib/main-shell.routes.ts [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
 
   {
-    path: 'forbidden',
-    loadChildren: () =>
-      import('@angular-challenges/module-to-standalone/forbidden').then(
-        (m) => m.ForbiddenModule,
-      ),
+    path: 'admin',
+    loadComponent: () =>
+      import('@angular-challenges/module-to-standalone/forbidden'),
```
Comment (tomalaforge) [inline libs/module-to-standalone/shell/src/lib/main-shell.routes.ts:28]: 👍

File: libs/module-to-standalone/user/shell/src/lib/user-shell.component.ts [hunk 2]
```diff
@@ -27,6 +28,8 @@ import { Component, Inject } from '@angular/core';
   host: {
     class: 'flex flex-col p-4 gap-3 border border-blue',
   },
+  standalone: true,
+  imports: [RouterLink, RouterOutlet],
 })
 export class UserShellComponent {
   constructor(@Inject(TOKEN) public token: string) {}
```
Comment (tomalaforge) [inline libs/module-to-standalone/user/shell/src/lib/user-shell.component.ts:35]: you should use the new inject function, especially for replacing @Inject(), this way you have the typing.

**Comment by tomalaforge** [comment]
Perfect, did you use the schematics? 
Did you know standalone and nx before?

**Comment by LMFinney** [comment]
> Perfect, did you use the schematics? Did you know standalone and nx before?

Yes, I used the schematics. And I've been using standalone and nx for a while.

**Comment by tomalaforge** [comment]
> > Perfect, did you use the schematics? Did you know standalone and nx before?
> 
> Yes, I used the schematics. And I've been using standalone and nx for a while.

ok it's what I thought, Because if you are not used to NX, this challenge is not very simple.

**Pull request closed without merging**